### ISSUE TITLE
Missing SC2166 URL in summary section

### ISSUE THREAD

**Issue by motminh** [koalaman/shellcheck#2983]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2166
- My shellcheck version (`shellcheck --version` or "online"): 0.9.0
- [ ] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [ ] I tried on https://www.shellcheck.net/ and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

In foo.rc line 86:
    if [ -n "${FOO_OBJ_PATH}" -a ! -d "${FOO_OBJ_PATH}" ]; then
                               ^-- SC2166 (warning): Prefer [ p ] && [ q ] as [ p -a q ] is not well defined.

```

#### Here's what shellcheck currently says:

shellcheck threw a SC2166, but the summary didn't list the wiki URL for reference and explanation:

For more information:
  https://www.shellcheck.net/wiki/SC2148 -- Tips depend on target shell and y...
  https://www.shellcheck.net/wiki/SC1090 -- ShellCheck can't follow non-const...
  https://www.shellcheck.net/wiki/SC2010 -- Don't use ls | grep. Use a glob o...

#### Here's what I wanted or expected to see:

https://www.shellcheck.net/wiki/SC2166 -- Prefer [ p ] && [ q ] as [ p -a q ] is not well defined.

**Comment by brother** [comment]
0.10.0 does show it.
![image](https://github.com/koalaman/shellcheck/assets/243351/34387849-e3b8-494e-8be9-232390045d1c)
